SR2021/2021-01-19-league-1-absent.md

```diff
@@ -0,0 +1,29 @@
+---
+to: Teams who didn't attend session 1
+subject: Student Robotics League 1
```

Comment:
Is this really the subject of the email?
Would something like "Helping you get the most out of your robot" be more accurate?

Reply:
Yeah i'm not super happy with this one, but really the email is about the fact they missed the league, so just "getting the most" isn't quite enough.

Reply:
Yeah, I appreciate that. I'm concerned that this could be read either (misread) as being about the next one (and thus ignored for a while) and/or just ignored as being something which is out of date (if the recipient perhaps hasn't checked their inbox in a while).